serverless.yml

```diff
@@ -0,0 +1,59 @@
+# This file is the main config file for your service.
+
+service: dmk-test-service
+custom:
+  service-name: dmk-test-service
+
+provider:
+  name: aws
+  runtime: python3.9
+  region: us-west-1
+  stage: ${opt:stage, 'dev'}
+  timeout: 180  # make sure there is time for an error state to be handled
+  apiGateway:
+    description: DMK Community Test service
+  environment:
+    DATADOG_API_KEY: ${ssm:/${self:custom.service-name}/${self:provider.stage}/datadog-api-key}
+    DATADOG_APP_KEY: ${ssm:/${self:custom.service-name}/${self:provider.stage}/datadog-app-key}
+    DMK_TEST_SERVICE_LAMBDA_KEY: ${ssm:/${self:custom.service-name}/service-api-key}
+    LOG_LEVEL: 'INFO'
+    SERVICE: ${self:custom.service-name}
+    STAGE: ${self:provider.stage}
+  iam:
+    role:
+      statements:
+        - Effect: 'Allow'
+          Action:
+            - 's3:GetObject'
+          Resource:
+            - 'arn:aws:s3:::dwelo-config/*'
+  vpc:
+    subnetIds:
+      - ${ssm:/${self:custom.service-name}/${self:provider.stage}/subnet-id1}
+      - ${ssm:/${self:custom.service-name}/${self:provider.stage}/subnet-id2}
+    securityGroupIds:
+      - ${ssm:/${self:custom.service-name}/${self:provider.stage}/security-group-id1}
+  tags:
+    service: ${self:custom.service-name}
+    stage: ${self:provider.stage}
+
+package:
+  patterns:
+    - '!test/**'
+    - '!venv/**'
+    - '!.travis.yml'
+    - '!README.md'
+
+functions:
+  function:
+    handler: src/lambda_function.lambda_handler
+    awsKmsKeyArn: arn:aws:kms:us-west-1:790585139892:key/f9187466-7568-4816-a7cf-6f41a39e1569   # cloud-service-lambda-key, needed to decrypt env vars
+    layers:
+      - arn:aws:lambda:us-west-1:464622532012:layer:Datadog-Python36-metric:2
+      - arn:aws:lambda:us-west-1:790585139892:layer:DweloCommonLambdas:56
+    events:
+      - eventBridge:
+          enabled: false
+          schedule: rate(10 minutes)
+          input:
+            somekey: somevalue
```

Comment:
does the input matter in this case because if it doesn't it would be nice if it was a small input like `o: k` or something like that but honestly it is a minor thing in the grand scheme of things.